### PR TITLE
Route abbrev

### LINKED ISSUE #225: Use RouteAbbreviation instead of ShortName?
Perhaps we should switch to using `<RouteAbbreviation>` instead of `<ShortName>`? We're currently using names that the public might not be familiar with. For example:

| short name | common name |
| --- | --- |
| 914 | R14E |
| 920 | P20E |
| 921 | P21E |
| R52 | TT |

RouteAbbreviation is used at https://bustracker.pvta.com.

This came up while looking at #220.

### PR DESCRIPTION
**Pull request by petermarathas**
Closes #225 

Routes can still be searched by their short name, but the route abbreviation is used for display. Note also that route-list-factory still sorts routes by number using short name.

### CODE REVIEW

**Comment by akaplo** [comment]
@petermarathas This seems right, but there's one issue: it will fail to properly display any already saved Favorite Routes (because they don't have the RouteAbbreviation saved), or any cached Routes in the `Routes and Stops` page (for the same reason).

Can you think of any way this can be avoided?  (I can think of one: use a ternary, like `route.RouteAbbreaviation ? route.RouteAbbreviation : route.ShortName`, because we know we'll have ShortName if we're dealing with cached data that doesn't have abbreviation.  After enough time, we could remove this after everyone has migrated).

screen shot 2016-09-28 at 11 15 59 am

**Comment by akaplo** [comment]
CC failures are OK.

**Review by akaplo — CHANGES_REQUESTED** [review]
See my main comment.

**Comment by akaplo** [comment]
@petermarathas and I talked about a solution to the aforementioned issue a few days ago.

File: www/pages/my-buses/my-buses-controller.js [hunk 1]
```diff
@@ -73,9 +73,18 @@ angular.module('pvta.controllers').controller('MyBusesController', function ($sc
   }
 
   var reload = function () {
-    localforage.getItem('favoriteRoutes', function (err, value) {
-      $scope.routes = value;
-      filterAlerts($scope.routes, Messages.query().$promise);
+    localforage.getItem('updatedRoutes', function(err, updated) {
```
Comment (petermarathas) [inline www/pages/my-buses/my-buses-controller.js]: @akaplo This gets the job done, checks whenever my-buses is reloaded

**Review by akaplo — APPROVED** [review]

**Pull request merged**